src/domain/Formats/Record/Record.tsx

```diff
@@ -61,13 +65,16 @@ class Record extends React.PureComponent <IRecordProps> {
   get label (): JSX.Element {
     const {
       name,
-      componentContext
+      componentContext,
+      variant
     } = this.props
 
     return (
       <FieldLabelWrapper
         data-component-type={Props.ComponentType.RecordName}
         data-component-context={componentContext}
+        hasLeftMargin={variant === RecordVariant.LabelRight}
```

Comment:
Shouldn't this just take `variant` as a prop and then change the CSS as it needs to?